HealthcareSystem/PatientWebApp/Controllers/ExaminationController.cs

```diff
@@ -65,5 +65,76 @@ public ActionResult AdvanceSearchExaminations(ExaminationSearchDTO examinationSe
 
         }
 
+        /// <summary>
+        /// /getting canceled examinations linked to a certain patient
+        /// </summary>
+        /// <param name="patientJmbg">patients jmbg</param>
+        /// <returns>if alright returns code 200(Ok), if patuentJmbg is null returns 400, if connection lost returns 500</returns>
+        [HttpGet("{canceled/patientJmbg}")]
+        public ActionResult GetCanceledExaminationsByPatient(string patientJmbg)
+        {
```

Comment:
Shouldn't it be "cancelled/{patientJmbg}"? I think you accidentally put the bracket in the wrong place.

Reply:
Yes. I made a mistake at one place, and just copied it.

Reply:
Fixed in af1b49d1baa111f3b0ee80cf8348c9c52a4adcae.